interp/interpreter.go

```diff
@@ -106,6 +103,17 @@ func (r *runner) run(fn *function, params []value, parentMem *memoryView, indent
 		}
 		switch inst.opcode {
 		case llvm.Ret:
+			t1 := time.Since(r.start)
+			if t1-t0 > time.Second {
+				// Provide some breadcrumbs for user trying to find their slow init functions.
+				fmt.Fprintln(os.Stderr, "interp: slow: startms", int(t0.Milliseconds()), "endms", int(t1.Milliseconds()), "func", fn.name)
```

Comment:
Hmm, IMHO this is a bit ugly. Perhaps it would have been better to store this information in the error object so it can be printed in `printCompilerError` in the main package?

Reply:
But it's not an error.  It's a diagnostic aid that gives the user early warning that they have a slow init, and where to look for it.

We could hide it behind a verbose flag, though.

Reply:
Yeah, I feel strongly that we shouldn't do any logging of this kind without an opt-in (via a flag for example). Normally we don't print anything for successful commands unless explicitly asked to (such as with `-size=short`), following Unix tradition. And I think that's a good tradition.